src/validators/uv_check.py

```diff
@@ -0,0 +1,67 @@
+"""UV Check Validator."""
+
+import logging
+from typing import List, Tuple
+
+import bpy
+
+from . import BaseValidator, Category
+
+
+def _get_multiple_uv_models(models: List[str]) -> List[str]:
+    return [
+        model for model in models
+        if (
+            bpy.data.objects[model].data is not None
+            and hasattr(bpy.data.objects[model].data, 'uv_layers')  # noqa: WPS421
+            and len(bpy.data.objects[model].data.uv_layers) > 1  # noqa: WPS219
+        )
+    ]
+
+
+def fix_uv_layers(export_data: dict):
+    """Remove all inactive UV layers from export data.
+
+    Parameters:
+        export_data: dict containing objects to be uploaded info
+
+    """
+    models = export_data.get('models', [])
+    multiple_uv_models = _get_multiple_uv_models(models)
+    for model in multiple_uv_models:
+        model_data = bpy.data.objects[model]
+        uv_layers = model_data.data.uv_layers
+        unwanted_uvs = [
+            uv for uv in uv_layers
+            if uv != uv_layers.active
```

Comment:
Aqui talvez não seja o fix correto: a UV com `active` True é a que está selecionada no painel de UVs, não a que tem a câmera selecionada (`active_render`). No caso da demo que você fez, deleta todas menos a última criada. Penso nisso porque na maioria dos problemas que eu corrigi com UVs, a UV certa era a com o ícone da câmera.

Reply:
E não deveria ser `if not uv_layers.active`?

Reply:
Hmmm bom saber! Vou modificar para pegar a com a câmera então, valeu!

> E não deveria ser if not uv_layers.active?
uv_layers tem uma lista de UVs e também um atributo `active` que já aponta pra ativa. No caso eu estou criando a `unwantend_uvs` passando por todas na lista, se elas não são a ativa, então parece certo